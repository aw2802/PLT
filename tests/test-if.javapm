public class TestIf{
  public void main(){
    print(111);

    if (true){
      print(1);
    }
    else{
      print(0);
    }

    print("Wahoo");
  }
}
