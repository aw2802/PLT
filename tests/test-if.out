111
1
0
000